docs/data/migration/migration-charts-v7/migration-charts-v7.md

```diff
@@ -138,6 +143,24 @@ Renames `LegendPosition` to `Position`.
 
 The `getSeriesToDisplay` function was removed in favor of the `useLegend` hook. You can check the [HTML Components example](/x/react-charts/components/#html-components) for usage information.
 
+## Renaming tooltip slots and props
+
+The slots `popper`, `axisContent`, and `itemContent` have been replaced by the `tooltip` slot, which is now the single entry point to customize the tooltip.
+
+For consistency, the `tooltip` props have been replaced by the `slotProps.tooltip`.
+
+```diff
+ <LineChart
+-   tooltip={{ trigger: 'item' }}
++   slotProps={{ tooltip: { trigger: 'item' }}}
+ />
+```
+
+Some helpers are provided to create your custom tooltip:
+
+- To override the **tooltip content**, use the `useItemTooltip` or `useAxisTooltip` to get the data, and wrap your component in `ChartsTooltipContainer` to follow the pointer position.
+- To override the **tooltip placement**, use the `ChartsItemTooltipContent` or `ChartsItemTooltipContent` to get the default data display, and place them in your custom tooltip.
```

Comment:
The two alternatives are the same: `ChartsItemTooltipContent`.

Reply:
```suggestion
- To override the **tooltip placement**, use the `ChartsAxisTooltipContent` or `ChartsItemTooltipContent` to get the default data display, and place them in your custom tooltip.
```